### PR TITLE
Fix/update links to dbt learn

### PR DESCRIPTION
**Pull request by coapacetic**
## What are you changing in this pull request and why?
The training and web team recently launched the new LMS platform over at learn.getdbt.com. This is a PR to start the process of updating all of the courses.getdbt.com/ links to learn.getdbt.com.

## Checklist
- [X] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [X] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Double check that all modified links actually work -- battle test it
- [x] Get full approval from the web team to hit send on this 🚀

### CODE REVIEW

**Comment by mirnawong1** [comment]
thanks @coapacetic  this looks great and the links word for me in the [staging link](https://docs-getdbt-nlw1uruao-dbt-labs.vercel.app/guides/core-to-cloud-3?step=9) 

❓ i think it might also be beneficial to add a redirect here: https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/vercel.json as well. _however_, i believe we have reached our limit on adding redirects. i don't want to block this pr but wanted to flag this in case we can also raise that in a related ticket since you're working with the web team on this. cc @runleonarun @matthewshaver @nghi-ly 

also separately, i believe the SL tableau integration is ga now so [this course page](https://learn.getdbt.com/courses/tableau-querying-the-semantic-layer) might need the 'beta' removed.
Screenshot 2024-05-23 at 11 15 41

**Comment by coapacetic** [comment]
Thanks @mirnawong1 -- appreciate the super quick review! 🙌 

re: Tableau BETA course -- great catch, I'll loop in @MichelleBaird-DBT to tackle that one.

re: redirects, I'll yield to the docs / web team on this one with the deeper understanding of how we use redirects.

related: I'll make sure we sync with Joey on migration tactics here so we merge this at the right time.

🚀

**Comment by mirnawong1** [comment]
you got it! will revert back!

**Comment by mirnawong1** [comment]
@coapacetic i'll ask the web team and can you let us now when you'd like this merged? thank you!

**Comment by coapacetic** [comment]
@mirnawong1 sounds great on the redirects -- I'll swing back here on when we actually hit the merge button.  Also tagged you in a slack thread for viz

**Comment by joeygaultdbt** [comment]
@coapacetic My recommendation here is to manually change the links within the content instead of using the redirect file while we figure out how to clean that one up and make it more efficient. This work will likely take a few weeks and I don't want to block your LMS work.

**Review by mirnawong1 — APPROVED** [review]
🚢 hey @coapacetic , approving this pr for you and the team to merge when you and @joeygaultdbt chat about the timeline

File: website/docs/guides/adapter-creation.md
Comment (matthewshaver) [inline website/docs/guides/adapter-creation.md]:
```suggestion
- The developer (and any product managers) ideally will have substantial experience as an end-user of dbt. If not, it is highly advised that you at least take the [dbt Fundamentals](https://learn.getdbt.com/courses/dbt-fundamentals) and [Advanced Materializations](https://learn.getdbt.com/courses/advanced-materializations) course.
```

File: website/docs/guides/redshift-qs.md
Comment (matthewshaver) [inline website/docs/guides/redshift-qs.md]:
```suggestion
Check out [dbt Fundamentals](https://learn.getdbt.com/courses/dbt-fundamentals) for free if you're interested in course learning with videos.
```

**Review by matthewshaver — APPROVED** [review]

File: website/blog/2023-03-30-guide-to-debug-in-jinja.md
Comment (coapacetic) [inline website/blog/2023-03-30-guide-to-debug-in-jinja.md]:
```suggestion
*Editor's note—this post assumes intermediate knowledge of Jinja and macros development in dbt. For an introduction to Jinja in dbt check out [the documentation](https://docs.getdbt.com/docs/build/jinja-macros) and the free self-serve course on [Jinja, Macros, Packages](https://learn.getdbt.com/courses/jinja-macros-and-packages).*
```

File: website/docs/best-practices/dbt-unity-catalog-best-practices.md
Comment (coapacetic) [inline website/docs/best-practices/dbt-unity-catalog-best-practices.md]:
```suggestion
- [Self-paced dbt fundamentals training course](https://learn.getdbt.com/courses/dbt-fundamentals)
```

File: website/docs/docs/build/build-metrics-intro.md
Comment (coapacetic) [inline website/docs/docs/build/build-metrics-intro.md]:
```suggestion
- [dbt Semantic Layer on-demand course](https://learn.getdbt.com/courses/semantic-layer)
```

File: website/docs/docs/cloud/about-cloud-setup.md
Comment (coapacetic) [inline website/docs/docs/cloud/about-cloud-setup.md]:
```suggestion
These settings are intended for dbt Cloud administrators. If you need a more detailed first-time setup guide for specific data platforms, read our [quickstart guides](/guides). If you want a more in-depth learning experience, we recommend taking the dbt Fundamentals on our [dbt Learn site](https://learn.getdbt.com/).
```

File: website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md
Comment (coapacetic) [inline website/docs/docs/use-dbt-semantic-layer/avail-sl-integrations.md]:
```suggestion
- [dbt Semantic Layer on-demand course](https://learn.getdbt.com/courses/semantic-layer)
```

File: website/docs/guides/azure-synapse-analytics-qs.md
Comment (coapacetic) [inline website/docs/guides/azure-synapse-analytics-qs.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/bigquery-qs.md
Comment (coapacetic) [inline website/docs/guides/bigquery-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/codespace-qs.md
Comment (coapacetic) [inline website/docs/guides/codespace-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/core-to-cloud-3.md
Comment (coapacetic) [inline website/docs/guides/core-to-cloud-3.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com) for on-demand video learning.
```

File: website/docs/guides/databricks-qs.md
Comment (coapacetic) [inline website/docs/guides/databricks-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/microsoft-fabric-qs.md
Comment (coapacetic) [inline website/docs/guides/microsoft-fabric-qs.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/productionize-your-dbt-databricks-project.md
Comment (coapacetic) [inline website/docs/guides/productionize-your-dbt-databricks-project.md]:
```suggestion
- [Advanced Deployment course](https://learn.getdbt.com/courses/advanced-deployment) if you want a deeper dive into these topics
```

File: website/docs/guides/redshift-qs.md
Comment (coapacetic) [inline website/docs/guides/redshift-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/sl-migration.md
Comment (coapacetic) [inline website/docs/guides/sl-migration.md]:
```suggestion
- [dbt Semantic Layer on-demand course](https://learn.getdbt.com/courses/semantic-layer)
```

File: website/docs/guides/snowflake-qs.md
Comment (coapacetic) [inline website/docs/guides/snowflake-qs.md]:
```suggestion
- Learn more with [dbt Learn courses](https://learn.getdbt.com)
```

File: website/docs/guides/starburst-galaxy-qs.md
Comment (coapacetic) [inline website/docs/guides/starburst-galaxy-qs.md]:
```suggestion
- [dbt Learn courses](https://learn.getdbt.com)
```

**Review by coapacetic — COMMENTED** [review]
final review for copy and links

**Comment by coapacetic** [comment]
heya @dbt-labs/product-docs team -- can I request one more 👍 on this after verifying links and making quick copy changes?

then we can merge away! thank you!

**Pull request merged**
